Storyline done

sinantheman opened the pull request:
## Description

- created comment headers for each wand file and wand.java
- merged sidequests story into general story we had before in storyline.md
- developed dialougue between Logoro and each world boss to be outputed before each boss battle in storyline.md
## General checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] I have updated `main` and merged to my branch before submitting pull request
- [ ] My pull request generate no conflicts with `main` branch
- [ ] I requested code review from other team members

sinantheman committed:
Crazy Amount of Changes

- method to access shop, inventory in Worlds.java

Co-Authored-By: Destroyer884 <114109307+Destroyer884@users.noreply.github.com>
Co-Authored-By: Fawaaz Kamali Siddiqui <86437242+thuglyfenoyan@users.noreply.github.com>

sinantheman committed:
Merge branch 'main' into sinan

sinantheman committed:
Merge branch 'main' into sinan

sinantheman committed:
Merge branch 'main' into sinan

sinantheman committed:
Merge branch 'main' into sinan

sinantheman committed:
Storyline completed

-finished comment headers for each wand file and wand.java
-merged sidequest storyline into general overall story in storyline.md
-developed dialougue to be outputed before each boss fight in storyline.md

sinantheman committed:
Updated storyline.md

-each boss is more evil, the player has more motivations to want to kill each boss beside the prism fragment
-no wand reward for killing the boss
-goal is to save the galaxy and logoro

fawaaz-kamali reviewed (commented):
Your changes will have lots of errors in the code. But I will still use your storyline.

sinantheman committed:
Merge branch 'main' into sinan

sinantheman committed:
fixed package errors

-restored wand packages

sinantheman committed:
Added storyline.md dialogue

-new dialogue for conversation between player and each world boss after the boss has been defeated

sinantheman committed:
Merge branch 'main' into sinan
